File: server/trash/bt.js
import noble from 'noble-winrt';

noble.on('stateChange', (state) => {
    if (state === 'poweredOn') {
        console.log("Scanning for devices...");
        noble.startScanning();
    } else {
        noble.stopScanning();
    }
});

noble.on('discover', (peripheral) => {
    console.log(`Found device: ${peripheral.advertisement.localName} - ${peripheral.id}`);
    noble.stopScanning();
});

File: server/trash/bt.py
import asyncio
from bleak import BleakScanner

async def scan_devices():
    print("Scanning for Bluetooth devices...")
    devices = await BleakScanner.discover()
    for device in devices:
        print(f"Found device: {device.name} - {device.address}")

asyncio.run(scan_devices())
